lib/fibonacci.rb

```diff
@@ -5,8 +5,20 @@
 # ....
 # e.g. 6th fibonacci number is 8
 
-# Time complexity: ?
-# Space complexity: ?
+# n is the input number
+# Time complexity: O(2n), since 2n steps will be needed.
```

Comment:
Incorrect. Each function call generates two additional function calls, so time complexity is really O(2^n).